src/metabase/driver/sql_jdbc/connection.clj

```diff
@@ -44,14 +45,15 @@
 
 (defmethod data-warehouse-connection-pool-properties :default
   [_]
-  {;; only fetch one new connection at a time, rather than batching fetches (default = 3 at a time). This is done in
+  { ;; only fetch one new connection at a time, rather than batching fetches (default = 3 at a time). This is done in
    ;; interest of minimizing memory consumption
    "acquireIncrement"             1
    ;; [From dox] Seconds a Connection can remain pooled but unused before being discarded.
    "maxIdleTime"                  (* 3 60 60) ; 3 hours
    "minPoolSize"                  1
    "initialPoolSize"              1
-   "maxPoolSize"                  15
+   "maxPoolSize"                  (or (config/config-int :mb-jdbc-data-warehouse-max-connection-pool-size)
```

Comment:
@camsaul any reason not to apply this same pattern and expose options can be easily defined as env vars for min/initial/idletime/etc?

Reply:
I've found myself using both of your "MB MAX" vars (app and warehouse), then later customizing the **app** db c3p0 properties via JAVA OPTS, but still not having a way to set the c3p0 properties for the **Warehouse** DBs.